wpiutil/src/llvm/raw_ostream.cpp

```diff
@@ -845,12 +845,12 @@ raw_null_ostream::~raw_null_ostream() {
 #endif
 }
 
-void raw_null_ostream::write_impl(const char *Ptr, size_t Size) {
+void raw_null_ostream::write_impl(const char * /*Ptr*/, size_t /*Size*/) {
```

Comment:
I did some testing. By commenting out the variable name, maintainers can be sure that variable is unused throughout the whole function. If a cast to void is used instead, there is no such guarantee because maintainers can use the variable without removing the void cast.